state/txn.go

```diff
@@ -1,3 +1,4 @@
+//nolint:forcetypeassert
```

Comment:
This package supports 0 error handling, so there is no point in refactoring the methods to support them in any way, since they're part of the legacy codebase